client/src/pages/DetailsPage/item-details.jsx

```diff
@@ -0,0 +1,54 @@
+import '../DetailsPage/item-details.scss';
+import GoBackButton from "../../components/GoBackButton/GoBackButton";
+import React, { useState } from 'react';
+
+export const ItemDetails = (props) => {
+	const [quantity, setQuantity] = useState(0);
+
+	const incrementQuantity = () => {
+		setQuantity(quantity + 1);
+	};
+
+	const decrementQuantity = () => {
+		if (quantity > 0) {
+			setQuantity(quantity - 1);
```

Comment:
Funktioniert zwar. Es ist allerdings zu bevorzugen einen funktionsbasierten Aufruf zu wählen.
`setQuantity(previousQuantity => Math.max(previousQuantity - 1, 0))`
`Math.max` gibt immer den größten Wert aus. So stellen wir sicher, dass der Wert immer 0 oder höher ist.
Das ist ein bisschen sicherer in der Funktionsweise.

Reply:
done!